submarine-server/server-submitter/submitter-yarnservice/src/main/java/org/apache/submarine/server/submitter/yarnservice/FileSystemOperations.java

```diff
@@ -179,9 +179,9 @@ public void validFileSize(String uri) throws IOException {
     } else {
       actualSizeByte = FileUtil.getDU(new File(uri));
     }
-    long maxFileSizeMB = submarineConfig
-        .getLong(SubmarineConfiguration.LOCALIZATION_MAX_ALLOWED_FILE_SIZE_MB,
-            SubmarineConfiguration.DEFAULT_MAX_ALLOWED_REMOTE_URI_SIZE_MB);
+    long maxFileSizeMB = submarineConfig.getLong(
+        SubmarineConfiguration.ConfVars.
+            SUBMARINE_LOCALIZATION_MAX_ALLOWED_FILE_SIZE_MB);
```

Comment:
Not need `DEFAULT_MAX_ALLOWED_REMOTE_URI_SIZE_MB` ?

Reply:
Yeah, add a method in SubmarineConfiguration:
```
public long getLong(ConfVars c) {
    return getLong(c.name(), c.getVarName(), c.getLongValue());
  }
```
This method will return the default value if the value is empty.